scripts/agent_step_receipt_relabeling_harness.py

```
@@ -0,0 +1,640 @@
+mutates user-facing statement fields and requires rejection.
+"""
+
+from __future__ import annotations
+
+import argparse
+import copy
+import hashlib
+import json
+import re
+import unicodedata
+from typing import Any, Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)